packages/integration-sdk-core/src/types/jobState.ts

```diff
@@ -4,7 +4,7 @@ export interface GraphObjectFilter {
   _type: string;
 }
 
-export type GraphObjectIteratee<T> = (obj: T) => void | Promise<void>;
+export type GraphObjectIteratee<T> = (obj: Readonly<T>) => void | Promise<void>;
```

Comment:
I'd consider this change a major version bump. Agreed?

Reply:
I've decided that since this enforcing an pre-existing understanding, it should only be a minor version bump.